### PR TITLE
new changes in branch

### PR DESCRIPTION
**Pull request by adityap7**
ok

### CODE REVIEW

File: sample.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # this is sample
 
-## file
+[Onyx Systems](https://en.wikipedia.org/wiki/Onyx_Systems) began selling early microcomputer-based Unix workstations in 1980. Later, [Sun Microsystems](https://en.wikipedia.org/wiki/Sun_Microsystems "Sun Microsystems"), founded as a spin-off of a student project at [Stanford University](https://en.wikipedia.org/wiki/Stanford_University "Stanford University"), also began selling Unix-based desktop workstations in 1982. While Sun workstations did not use commodity PC hardware, for which Linux was later originally developed, it represented the first successful commercial attempt at distributing a primarily single-user microcomputer that ran a Unix operating system
```
Comment (ramlaxman) [inline sample.md]: This para includes passive voice and wordy sentences,

Comment (adityap7) [reply sample.md]: ajun kahi

Comment (ramlaxman) [inline sample.md]:
```suggestion
In 1980, [Onyx Systems](https://en.wikipedia.org/wiki/Onyx_Systems) began selling early microcomputer-based Unix workstations. Later, [Sun Microsystems](https://en.wikipedia.org/wiki/Sun_Microsystems "Sun Microsystems"), founded as a spin-off of a student project at Stanford University, also began selling Unix-based desktop workstations in 1982. While Sun workstations did not use commodity PC hardware, for which Linux was later originally developed, it represented the first successful commercial attempt at distributing a primarily single-user microcomputer that ran a Unix operating system
```

Comment (ramlaxman) [reply sample.md]: Kahi nahi, evadh kar pahil. :joy:

**Pull request merged**